### PR TITLE
Release 2.0.0 alpha.5

### PR DESCRIPTION
**Pull request by farmaazon**
### Checklist
- [x] All checks passed
- [x] Changelog is checked.
- [x] QA approval

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+
+- [Delete key will delete selected nodes][1538].
+
+#### EnsoGL (rendering engine)
+
+#### Enso Compiler
+
+- [Updated Enso engine to version 0.2.11][1541].
+
+If you're interested in the enhancements and fixes made to the Enso compiler,
```
Comment (wdanilo) [inline CHANGELOG.md]: This changelog is not ready to be merged as:
1. it contains unused sections;
2. It does not describe the issues from the user's perspective (it should tell that there was an issue that only backspace was removing nodes and now delete does it as well).

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+you can find their release notes
+[here](https://github.com/enso-org/enso/blob/main/RELEASES.md).
+
+[1541]: https://github.com/enso-org/ide/pull/1511
+[1538]: https://github.com/enso-org/ide/pull/1538
```
Comment (wdanilo) [inline CHANGELOG.md]: Also, this PR seems bigger than just "adding delete key" - but it is not described in changelog. Probably its OK, but Id love to learn more of why :)

**Review by wdanilo — CHANGES_REQUESTED** [review]

**Comment by wdanilo** [comment]
What is the status of this?

File: CHANGELOG.md (hunk 2 above)
Comment (farmaazon) [reply CHANGELOG.md]: This is the change summary I see in delete PR, it seems reasonable:
image

Are you sure you looked at the proper PR?

Comment (farmaazon) [reply CHANGELOG.md]: This is the change summary I see in delete PR, it seems reasonable:
image

Are you sure you looked at the proper PR?

**Pull request merged**